.github/workflows/trunk.yml

```
@@ -157,6 +157,25 @@ jobs:
         source /opt/arm-sdk/setup_path.sh
         PYTHON_EXECUTABLE=python bash examples/arm/run.sh /opt/arm-sdk buck2
 
+  test-arm-reference-delegation:
+    name: test-arm-reference-delegation
+    uses: pytorch/test-infra/.github/workflows/linux_job.yml@main
+    with:
+      runner: linux.2xlarge
+      docker-image: executorch-ubuntu-22.04-arm-sdk
+      submodules: 'true'
+      ref: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.sha || github.sha }}
+      script: |
+        # The generic Linux job chooses to use base env, not the one setup by the image
+        CONDA_ENV=$(conda env list --json | jq -r ".envs | .[-1]")
+        conda activate "${CONDA_ENV}"
+
+        source .ci/scripts/utils.sh
+        install_flatc_from_source
+        install_executorch
+
+        PYTHON_EXECUTABLE=python bash backends/arm/test/arm_tosa_reference.py
```

This looks like a typo, I think you mean

```suggestion
        PYTHON_EXECUTABLE=python bash backends/arm/test/arm_tosa_reference.sh
```

---

ah, yes.